docs/user_stories/debug_failed_build.md

```diff
@@ -0,0 +1,23 @@
+### As a developer, with a PR that fails to build, I want to find the root cause so I can fix it and merges
+#### User Journey
+1. GitHub notifies user that build failed
+2. User accesses logs
+3. User looks for a stacktrace with their App’s processes name
+4. User finds stacktrace + root cause and begins to fix
+5. User commits fix
+6. User’s Build passes
+7. User merges PR
+#### Use Case
+1. GitHub has Status update and comment from fastlane on PR indicating that the build failed
+2. User opens fastlane ci Web UI
+4. Assuming already logged in
```

Comment:
we should move preconditions into a `preconditions` section or more informally: `Assumptions`, take a look at our use cases in how we do it: https://github.com/fastlane/ci/blob/master/docs/use_cases/add_project_for_pr_status.md

Reply:
I was hoping to fit all the characteristic of the story into the story statement itself, but I don't think that's reasonable. Will be adding an Assumptions section.